python/paddle/utils/preprocess_img.py

```diff
@@ -127,7 +127,7 @@ def create_dataset_from_list(self, path):
             image_path = items[0]
             label_name = items[1]
             if not label_name in label_set:
-                label_set[label_name] = len(label_set.keys())
+                label_set[label_name] = len(list(label_set.keys()))
```

Comment:
This line is not necessary.

Reply:
actually, it is, keys will return a iterator in python3, we should case it to list here

Reply:
we can invoke `len` to `dict.keys()` in Python3. And it does not return an iterator in Py3.

```
Python 3.5.2 (default, Nov 17 2016, 17:05:23) 
[GCC 5.4.0 20160609] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> a = {1:2, 3:4}
>>> len(a.keys())
2
>>> type(a.keys())
<class 'dict_keys'>
>>> 
```